torch/ao/quantization/pt2e/_numeric_debugger.py

```diff
@@ -31,7 +31,7 @@ def generate_numeric_debug_handle(ep: ExportedProgram) -> None:
         generate_numeric_debug_handle(ep)
 
         m = ep.module()
-        quantizer = XNNPACKQuantizer()
+        quantizer = XNNPACKQuantizer() # In the ExecuTorch repo.
```

Comment:
please add an import for this

Reply:
what do you mean? This is an example, do you mean to put import line in addition to the comment in the doc string?

Reply:
yeah.. just to make it clearer since it's in executorch now